### PR TITLE
Update webrender.

### PR DESCRIPTION
**Pull request by jdm**

### CODE REVIEW

File: components/webgpu/Cargo.toml [hunk 1]
```diff
@@ -17,5 +17,5 @@ log = "0.4"
 malloc_size_of = { path = "../malloc_size_of" }
 serde = "1.0"
 servo_config = {path = "../config"}
-smallvec = "0.6"
+smallvec = { version = "0.6", features = ["serde"] }
```
Comment (atouchet) [inline components/webgpu/Cargo.toml:20]: Could smallvec be updated to 1.x in these components as well?

Comment (jdm) [reply components/webgpu/Cargo.toml:20]: I'd pretty to make that change separately.

File: Cargo.lock [hunk 2]
```diff
@@ -6331,22 +6353,24 @@ dependencies = [
 [[package]]
 name = "webrender"
 version = "0.61.0"
-source = "git+https://github.com/servo/webrender#4ba17a61580a03b68881ffba7ee22bcf8e1aac72"
+source = "git+https://github.com/servo/webrender#5a5cb7951886977254164b967008177aadc275d0"
```
Comment (atouchet) [inline Cargo.lock:6356]: Are you deliberately using an older git version? It would be nice to have the latest release to get the dwrote update.

**Comment by SimonSapin** [comment]
Diff looks good but the build is failing with `error[E0277]: the trait bound `webrender_api::api::ScrollClamping: serde::Deserialize<'_>` is not satisfied` and other errors: https://community-tc.services.mozilla.com/tasks/EY03XjQdTuOwWxXxER_qdQ/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FEY03XjQdTuOwWxXxER_qdQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L2049

**Comment by jdm** [comment]
Updating to a more recent revision fixed the missing serialize problem.

File: Cargo.toml [hunk 3]
```diff
@@ -33,3 +33,7 @@ winapi = { git = "https://github.com/servo/winapi-rs", branch = "patch-1" }
 spirv_cross = { git = "https://github.com/servo/spirv_cross", branch = "wgpu-servo" }
 surfman-chains = { git = "https://github.com/asajeffrey/surfman-chains" }
 surfman = { git = "https://github.com/servo/surfman" }
+
+[patch."https://github.com/servo/webrender"]
+webrender = { git = "https://github.com/jdm/webrender/", branch = "serialize" }
```
Comment (atouchet) [inline Cargo.toml:38]: Do these not need to be changed back?

Comment (jdm) [reply Cargo.toml:38]: Ugh, I think I've been testing the wrong branch :(

**Pull request merged**